ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -1,3 +1,30 @@
+- scm:
+    name: ceph-build
```

Comment:
You shouldn't need to clone ceph-build. Just use ``!include-raw`` with the shell builders below.

See: https://github.com/ceph/ceph-build/blob/master/ceph-build/config/definitions/ceph-build.yml#L56